sklearn/preprocessing/_data.py

```diff
@@ -613,8 +613,10 @@ def minmax_scale(X, feature_range=(0, 1), *, axis=0, copy=True):
         otherwise (if 1) scale each sample.
 
     copy : bool, default=True
-        Set to False to perform inplace scaling and avoid a copy (if the input
-        is already a numpy array).
+        If False, try to avoid a copy and do inplace scaling instead.
```

Comment:
Some wording tweaks, not sure if they are significantly better. I am pretty sure though that it should be "in place", not inplace
```suggestion
        If False, try to avoid a copy and perform scaling in place.
```
```suggestion
        If False, try to avoid a copy and scale in place.
```